src/status_im/contexts/onboarding/sync_or_recover_profile/view.cljs

```diff
@@ -0,0 +1,117 @@
+(ns status-im.contexts.onboarding.sync-or-recover-profile.view
+  (:require
+    [quo.core :as quo]
+    re-frame.db
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [status-im.common.resources :as resources]
+    [status-im.config :as config]
+    [status-im.contexts.onboarding.new-to-status.style :as style]
+    [utils.debounce :as debounce]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn sign-in-options
+  []
+  (let [window (rf/sub [:dimensions/window])]
+    [rn/view {:style style/options-container}
+     [quo/text
+      {:style  style/title
+       :size   :heading-1
+       :weight :semi-bold}
+      (i18n/label :t/sync-or-recover-profile)]
+     [quo/small-option-card
+      {:variant      :main
+       :title        (i18n/label :t/sign-in-by-syncing)
+       :subtitle     (i18n/label :t/if-you-have-status-on-another-device)
+       :button-label (i18n/label :t/scan-sync-code)
+       :image        (resources/get-image :generate-keys)
+       :max-height   (- (:height window)
```

Comment:
> How did you arrive at 56 or 220?

sync recover view is copied from new to status view since they have the same layout

![CleanShot 2024-06-11 at 15 39 53](https://github.com/status-im/status-mobile/assets/15090582/26102b9e-2272-4e5d-a35d-b862d196d5b3)

![CleanShot 2024-06-11 at 15 39 49](https://github.com/status-im/status-mobile/assets/15090582/248e2a28-ad51-4799-853f-ed7c0cb936d2)